orangecontrib/timeseries/widgets/owspiralogram.py

```diff
@@ -176,11 +284,13 @@ def selection_indices(self, indices):
         chart=dict(
             type='column',
             polar=True,
-            panning=False,  # Fixes: https://github.com/highcharts/highcharts/issues/5240
+            panning=False,
+            # Fixes: https://github.com/highcharts/highcharts/issues/5240
```

Comment:
If you wouldn't mind another round of editing, you can move this comment above `panning=False`. Comments refer to the next line.

It's not an issue because anybody can click a link and see it's about panning, not selecting.